server/issue.go

```diff
@@ -846,12 +846,12 @@ func (p *Plugin) UnassignIssue(instance Instance, mattermostUserID types.ID, iss
 	// check for valid issue key
 	_, err = client.GetIssue(issueKey, nil)
 	if err != nil {
-		return "", errors.Errorf("We couldn't find the issue key `%s`.  Please confirm the issue key and try again.", issueKey)
+		return "", errors.Errorf("We couldn't find the issue key `%s`. Please confirm the issue key and try again.", issueKey) //nolint:revive
```

Comment:
Can we somehow disable this exact case of `revive`'s check rather than putting these `nolint`s everywhere?

Reply:
Good suggestion, added that